noir_stdlib/src/bigint.nr

```diff
@@ -0,0 +1,53 @@
+use crate::ops::{Add, Sub, Mul, Div, Rem,};
+
+struct BigInt {
+    pointer: u32,
+    modulus: u32,
+}
+
+impl BigInt {
+    #[builtin(bigint_add)]
+    pub fn bigint_add(_self: Self, _other: BigInt) -> BigInt {
+    }
+    #[builtin(bigint_neg)]
+    pub fn bigint_neg(_self: Self, _other: BigInt) -> BigInt {
+    }
+    #[builtin(bigint_mul)]
+    pub fn bigint_mul(_self: Self, _other: BigInt) -> BigInt {
+    }
+    #[builtin(bigint_div)]
+    pub fn bigint_div(_self: Self, _other: BigInt) -> BigInt {
+    }
+    #[builtin(bigint_from_le_bytes)]
+    pub fn from_le_bytes(_bytes: [u8], _modulus: [u8]) -> BigInt {}
+    #[builtin(bigint_to_le_bytes)]
+    pub fn to_le_bytes(_self: Self) -> [u8] {}
```

Comment:
```suggestion
    #[builtin(bigint_add)]
    pub fn bigint_add(self, other: BigInt) -> BigInt {
    }
    #[builtin(bigint_neg)]
    pub fn bigint_neg(self, other: BigInt) -> BigInt {
    }
    #[builtin(bigint_mul)]
    pub fn bigint_mul(self, other: BigInt) -> BigInt {
    }
    #[builtin(bigint_div)]
    pub fn bigint_div(self, other: BigInt) -> BigInt {
    }
    #[builtin(bigint_from_le_bytes)]
    pub fn from_le_bytes(bytes: [u8], modulus: [u8]) -> BigInt {}
    #[builtin(bigint_to_le_bytes)]
    pub fn to_le_bytes(self) -> [u8] {}
```

Reply:
Won't this generate unused warnings in stdlib when we reenable the import for bigint?

Reply:
https://github.com/noir-lang/noir/pull/4179

Reply:
nice!